README.md

```diff
@@ -124,3 +124,45 @@ Once the Ziti Tunneler SDK is initialized with a network device and ziti-sdk
 callbacks, a tunneler application only needs to indiciate which service(s)
 that should be  
 
+## Multi-Platform Linux Builder Container
```

Comment:
i'd personally like it better if this information was moved to a "BUILDING.md" and then referenced herein. in building.md we'd include simple instructions:
```
mkdir -p build
cd build
cmake .. && cmake --build .
```
for example... and then i also would expect to see the prerequisites covered (not how to install them though) and include all major os mac/linux/windows/ and add in 'docker' here